scripts/pylint_extensions.py

```diff
@@ -1424,20 +1428,23 @@ def process_module(self, node):
             triple_quote_counter += line.count(multi_line_indicator)
 
             if line.endswith(b'"""') and triple_quote_counter == 2:
-                temp_line_num = line_num
-                while file_content[temp_line_num + 1] == b'\n':
-                    empty_line_counter += 1
-                    temp_line_num += 1
-                if empty_line_counter > 1:
-                    self.add_message(
-                        'only-a-single-empty-line-should-be-provided',
-                        line=line_num + 1)
-                    break
-                if empty_line_counter == 0:
-                    self.add_message(
-                        'no-empty-line-provided-below-fileoverview',
-                        line=line_num + 1)
-                    break
+                closing_line_index_of_fileoverview = line_num
+                break
+
+        empty_line_check_index = closing_line_index_of_fileoverview
+        if empty_line_check_index < file_length:
```

Comment:
Still careless -- I think there's an off-by-one error.

I suggest that you add a backend test where there's no newline after the fileoverview string to verify that the behaviour in that case is correct.

Reply:
Yeah @seanlip, I tested the behavior by making file overview the last line.
hence we require `empty_line_check_index` to be less than `file_length-1`.

Reply:
done